Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/patientlocationquery/adapter/dao/PatientLocationQueryDaoIntTest.java

```diff
@@ -56,4 +61,28 @@ public void testGetAllByRLSId() {
         assertEquals(0, result.size());
     }
 
+    @Test
+    public void testPLQResponseOutbound() {
```

Comment:
If youre adding new entries into the DB, clean them up so they do not persist and potentially break other tests.
Since we have no transaction management, you cannot just rollback the changes so you will need to delete these manually.